modules/apps/object/object-test/src/testFunctional/tests/ObjectAutoGeneratedLayout.testcase

```diff
@@ -104,8 +104,6 @@ definition {
 	@description = "LPS-143065 - Verify that the file can be downloaded (a download button appears on the side of the name on the mouse hover)"
 	@priority = "4"
 	test CanDownloadFile {
-		property portal.upstream = "quarantine";
-
 		ObjectAdmin.addObjectViaAPI(
 			labelName = "Custom Object 151581",
 			objectName = "CustomObject151581",
```

Comment:
Hey @thaiscabrals was this test fixed as well? I didn't see any related changes to this so wasn't sure if this was intentional

Reply:
Hi @timpak, I had already linked in LPS-147989, commented and closed.
Next time, I will add the link to the description. Thanks for help.